pyproject.toml

```diff
@@ -1,9 +1,9 @@
 [project]
 name = "ctakesclient"
-requires-python = ">= 3.7"
+requires-python = ">= 3.8"
```

Comment:
it's walrus time baybee

Reply:
Oh shoot you're right -- I thought it was in 3.9. This means the ETL can also use walruses.